jwql/website/apps/jwql/urls.py

```diff
@@ -74,6 +74,10 @@
     path('about/', views.about, name='about'),
     path('dashboard/', views.dashboard, name='dashboard'),
     path('edb/', views.engineering_database, name='edb'),
+    path('query_anomaly/', views.query_anomaly, name='query_anomaly'),
+    path('query_anomaly_2/', views.query_anomaly_2, name='query_anomaly_2'),
+    path('query_anomaly_3/', views.query_anomaly_3, name='query_anomaly_3'),
+    path('query_submit/', views.query_submit, name='query_submit'),
```

Comment:
These all look and work great in the browser, but this again points out to me that there is likely a single form/HTML page solution out there, instead of having to divide things up in multiple pages/forms.  Something to think about in the future 🤔